phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexTool.java

```diff
@@ -641,18 +668,27 @@ public Job getJob() {
         return job;
     }
 
-    private void createIndexToolOutputTable(Connection connection) throws Exception {
+    private void createIndexToolTables(Connection connection) throws Exception {
         ConnectionQueryServices queryServices = connection.unwrap(PhoenixConnection.class).getQueryServices();
         Admin admin = queryServices.getAdmin();
-        if (admin.tableExists(TableName.valueOf(OUTPUT_TABLE_NAME))) {
-            return;
+        if (!admin.tableExists(TableName.valueOf(OUTPUT_TABLE_NAME))) {
```

Comment:
will this work on a cluster with namespaces enabled?

Reply:
It should work. Tables can be created without specifying a namespace. Then they are created within the default namespace and the default namespace is not specified in commands/operations as far as I understand.